scripts/check_crlf.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+    Checker for line endings
+    ~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Make sure each Python does not use Windows-style file endings.
+
+    :copyright: Copyright 2006-2020 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import sys
+import os
+
+if __name__ == '__main__':
+    for directory in sys.argv[1:]:
+        if not os.path.exists(directory):
+            continue
+
+        for root, dirs, files in os.walk(directory):
+            for filename in files:
+                if not filename.endswith('.py'):

```

Comment:
hm, this is only checking .py files but the problem was in the bashcomp file...

Reply:
As far as I can tell if I mess up the CR/LF I mess it up everywhere. I can exclude `*.pyc`, I thought this was fairly safe for the particular issue at hand.

Reply:
Added `.bashcomp`.

Reply:
Makes sense.